docs/howto/connect_service_with_elastic_stack.md

```diff
@@ -0,0 +1,73 @@
+# HOWTO: Connect the service with the Elastic stack
+
+_This method is a temporary workaround until a finest solution is delivered. Issue: [#190](https://github.com/elastic/elastic-package/issues/190)._
+
+While building or maintaining an integration it's convenient to keep the service connected to the same network as the Elastic
+stack instead of linking it temporarily for the execution of system tests. It's especially useful if you're adjusting Kibana
+dashboards or want to examine metrics data collected for the service in the Elasticsearch index.
+
+The procedure assumes an existence of two Docker networks - stack network and service network. The goal is to expose
+the stack network to the service container. The Elastic Agent will have an access to the service metrics endpoint and its
+logs (via mounted volume).
+
+## User guide (based on the Apache integration)
+
+1. If you haven't done it already, please boot up the Elastic stack:
+
+```bash
+elastic-package stack up -d -v
```

Comment:
Also let's make it more clear that the compose file is located at `~/.elastic-package/stack/snapshot.yml ` for those that are not familiar with the internals.

Reply:
I added the "Advanced" section to cover it, but in general this method assumes that you don't interact with `snapshot.yml`, but can use `docker` command to achieve the goal.